config/initializers/doorkeeper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+Doorkeeper.configure do
+  orm :active_record
+  base_controller "ActionController::Base"
+
+  # Enabled grant flows
+  grant_flows %w[client_credentials]
+
+  # Security
+  hash_token_secrets
+  authorization_code_expires_in 10.minutes
+  access_token_expires_in 2.hours
+  force_ssl_in_redirect_uri Manyfold::Application.config.force_ssl
```

Comment:
This'll prevent people developing apps on localhost against a running manyfold server, that's why we don't have it enabled in mastodon.

Reply:
Hm, good point.

Reply:
Logged as new issue #3964